File: multiples-of-three-and-five/multiplesOfThreeAndFive.js
const solution = (number) => {

    let output = 0;

    for(let i = number - 1; i >= 2; i--) {
        let checkThree = i / 3;
        let checkFive = i / 5;
        if(checkThree % 1 === 0) {
            output += i;
        } else 
        if(checkFive % 1 === 0) {
            output += i;
        };
    };
    return output;
};

//https://www.codewars.com/kata/514b92a657cdc65150000006/train/javascript

module.exports = { solution };

File: python-venv/multiples-of=three-and-five/multiplesOfThreeAndFive.py
def multiplesOfThreeAndFive(number):
    output = 0

    for num in range(number):
        checkThree = num / 3
        checkFive = num / 5
        if (checkThree).is_integer() == True or (checkFive).is_integer() == True:
            output += num

    return output